src/app/catalog/services/controller/controller.service.ts

```diff
@@ -124,4 +110,31 @@ export class ControllerService {
   displayCatalogFilterButton(isDisplayed: boolean): void {
     this.catalogFilterService.setFiltersButtonDisplayed(isDisplayed);
   }
+
+  filterEquipment(): void {
+    const payload = this.catalogFilterService.equipmentFilterRequest;
+    this.api.filterEquipment(payload).subscribe((res) => {
+      this.store.dispatch(new GetCatalog(res.items));
+    });
+  }
+
+  isFilteringApplied(): Observable<boolean> {
+    return this.catalogFilterService.getFilteringApplied();
+  }
+
+  set selectedCategoryId(categoryId: number) {
```

Comment:
as i could see all the getters and setters bellow run functions of catalog filter service, which write to store and get information form store. As I can see this service also have acces to the  stroe. so should we refer to filter service methods?